templates/questions/index.html.ep

```diff
@@ -91,6 +92,13 @@ $(document).ready(function() {
 			<img class="containedImg headerImg" src="/img/newQuestion.png" alt='new question' onclick="toggleQuestionForm()">
 		</div>
 		<div class="headerImgContainer accountButtonContainer" onclick="window.location = '/account'">
+			% if ($c->session('id')){
+        	<div class="Name"><%= $user->{first_name}%></div>
+			%}
+			%if ($c->session->{admin} == 1) {
+			<div class="Name">*</div>
```

Comment:
Rather than adding a text star to denote admin, I'd rather see you handle it with styling.  e.g.

```
<div class="Name" class="<%= $c->session->{admin} ? 'admin' : '' %>"><%= $user->{first_name}%></div>
```

Or something like that.  Does that make sense why this suggestion would be preferable?